content/components/relative-time.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Relative time
+status: Experimental
+---
+
+import {Box, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A relative time displays the relative time difference between the source date-time and the provided date-time.
```

Comment:
```suggestion
  A relative time displays the time difference between now and the specified date-time.
```

I hope this makes sense. I find the sentence a bit confusing otherwise and you repeat `relative time`.

Is the `source date-time` now or can it be a different time as well?

Reply:
Makes sense. The source date-time is the current time and that can't be modified. 

I'm thinking of using `current time` or `current timestamp` instead of `now`.

Reply:
Makes sense. The source date-time is the current time and that can't be modified. 

I'm thinking of using `current time` or `current timestamp` instead of `now`.

Reply:
`current time` is probably a good mix between not to technical, but still accurate.

Reply:
rephrasing it to avoid `time` repetition

```suggestion
  A relative time displays the time difference between the current time and the specified date-time
```